src/main/java/org/opentripplanner/middleware/otp/OtpRequestProcessor.java

```diff
@@ -0,0 +1,96 @@
+package org.opentripplanner.middleware.otp;
+
+import com.mongodb.MongoException;
+import org.eclipse.jetty.http.HttpStatus;
+import org.opentripplanner.middleware.models.TripRequest;
+import org.opentripplanner.middleware.models.TripSummary;
+import org.opentripplanner.middleware.models.OtpUser;
+import org.opentripplanner.middleware.otp.core.api.model.TripPlan;
+import org.opentripplanner.middleware.persistence.Persistence;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import static org.opentripplanner.middleware.utils.JsonUtils.logMessageAndHalt;
+
+public class OtpRequestProcessor {
+    private static final Logger LOG = LoggerFactory.getLogger(OtpRequestProcessor.class);
+    private static final String USER_ID = "userId";
+    private static final String BATCH_ID = "batchId";
+    private static final String FROM_PLACE = "fromPlace";
+    private static final String TO_PLACE = "toPlace";
+
+    public static String planning(Request request, spark.Response response, String otpServer, String endPoint) {
```

Comment:
Would prefer that args map to Spark.get and that we set the otpServer and endPoint another way.